sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/ConversionErrorCode.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion.Models
+{
+    /// <summary> The ConversionErrorCode. </summary>
+    public readonly partial struct ConversionErrorCode
+    {
+        /// <summary> UNKNOWN. </summary>
+        [CodeGenMember("Unknown")]
+        public static ConversionErrorCode Unknown { get; } = new ConversionErrorCode(UnknownValue);
+        /// <summary> NO_ERROR. </summary>
+        [CodeGenMember("NOError")]
+        public static ConversionErrorCode NoError { get; } = new ConversionErrorCode(NoErrorValue);
+        /// <summary> SERVICE_ERROR. </summary>
+        [CodeGenMember("ServiceError")]
+        public static ConversionErrorCode ServiceError { get; } = new ConversionErrorCode(ServiceErrorValue);
+        /// <summary> INVALID_ASSET_URI. </summary>
+        [CodeGenMember("InvalidAssetURI")]
+        public static ConversionErrorCode InvalidAssetUri { get; } = new ConversionErrorCode(InvalidAssetUriValue);
+        /// <summary> INVALID_JOB_ID. </summary>
+        [CodeGenMember("InvalidJOBID")]
```

Comment:
Why is this cased like this?

Reply:
So autogen for some reason turned INVALID_JOB_ID into InvalidJOBID when generating the extensible enum, I guess it thought anything with <4 letters in that uppercase underscore-delimited error code string was an acronym? The custom partial class was for renaming them with proper casing

Reply:
Fixed through swagger change